frontend/packages/dev-console/src/components/add/AddCardItem.tsx

```diff
@@ -47,13 +51,17 @@ const AddCardItem: React.FC<AddCardItemProps> = ({
       componentProps={{
         'data-test': `item ${id}`,
       }}
-      href={resolvedHref(href, namespace)}
+      href={href ? resolvedHref(href, namespace) : null}
       onClick={(e: React.SyntheticEvent) => {
         fireTelemetryEvent('Add Item Selected', {
           id,
           name: label,
         });
-        navigateTo(e, resolvedHref(href, namespace));
+        if (href) {
+          navigateTo(e, resolvedHref(href, namespace));
+        } else {
+          cta.callback({ namespace, toast });
```

Comment:
Better to add a check for `callback` as if it is been passed or not. As `callback` is an optional property.